app/views/runs/index.slim

```diff
@@ -21,6 +21,28 @@ article
     .col-lg-12
       - if logged_in?
         - if current_user.runs.any?
```

Comment:
Not a problem, but if you wanted to remove one level of indent for readability this `if` could be combined with the one above.

Reply:
They have different `else`s however. Might still clean this up in another branch later though, the indentation levels are getting out of hand.

Reply:
Ah, I didn't think to look for `else`s, I just noticed that the indentation was getting pretty nutty.